agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh

```diff
@@ -235,7 +237,11 @@ function install_k8s_agent {
     sed -i.bak -e "s|# serviceAccount: sysdig-agent|serviceAccount: sysdig-agent|" $DAEMONSET_FILE
 
     # Use IBM Cloud Container Registry instead of docker.io
-    sed -i.bak -e "s|\( *image: \)sysdig|\1icr.io/ext/sysdig|g" $DAEMONSET_FILE
+    if [ ! -z "$AGENT_VERSION" ]; then
```

Comment:
if you default the version to `latest` then you can remove the if block and just always replace with a version

Reply:
Perfect. My thoughts exactly. Fixed.